requirements.txt

```diff
@@ -1,3 +1,3 @@
 beautifulsoup4==4.9.1
-paramiko==2.7.2
+git+https://github.com/phantomcyber/paramiko.git@main #paramiko with sha256
```

Comment:
Use custom paramiko from phantomcyber, which is forked from the original repo. In this way, it's easy to maintain and better for apps sharing similar issues.